### ISSUE TITLE
'X-Frame-Options' to 'sameorigin' with CloudFront

### ISSUE THREAD

**Issue by WojciechowskiPiotr** [amazon-connect/amazon-connect-streams#352]
Hi,

I'm trying to follow the instructions provided in the documentation. I have S3 bucket behind the CloudFront, and I whitelisted the CloudFront URL on Amazon connect. But when opening the page I created (as per instruction) I get an error:

Refused to display ' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

I tried to whitelist the S3 bucket as well but with no change, but according to amazon-connect-streams documentation the whitelisting the CloudFront should be sufficient. I used HTTPS of course and embedded agent panel from the documentation example.

What am I missing? I'm using the latest 1.6.0 version of amazon-connect-streams. I see the documentation is not up to date as the whitelisting looks a little different on Amazon Connect now but I don't think thats the issue.

Thanks

**Comment by jauzepy** [comment]
Hello Piotr,
A few things: Make sure you are using the right ccpUrl(the one finishing with ccp-v2/ or ccp#/), make sure you are logged in your ccp in an other tab on the same browser (checkout #192 for details, logging in via iframe is not supported) and that you are not using "Incognito Window" as authentication token doesn't seems to work within the tabs in that mode. I got it working yesterday night via Cloudfront as well. Thanks

**Comment by WojciechowskiPiotr** [comment]
> Hello Piotr,
> A few things: Make sure you are using the right ccpUrl(the one finishing with ccp-v2/ or ccp#/), make sure you are logged in your ccp in an other tab on the same browser (checkout #192 for details, logging in via iframe is not supported) and that you are not using "Incognito Window" as authentication token doesn't seems to work within the tabs in that mode. I got it working yesterday night via Cloudfront as well. Thanks

Yes, it is working if you are logged in on another tab already. But why the login window itself is not working? Is there any workaround?

**Comment by WojciechowskiPiotr** [comment]
Hi,
Any workaround for this or is there any best practice recommendation on this problem?
Thanks

**Comment by annjawn** [comment]
Facing the same issue even with the domains added as allowed origin. I've tried almost all the suggested solutions in all the open and closed issues in this repo but none of those seem to work. For me it initially looked like it worked with `ccp-v2/`, but the login popup would open, and once I logged in, the whole CCP console would open up in the popup itself; is this expected behavior? Is it supposed to open in a new tab or is it supposed to load inside of the iframe? I can't tell since I haven't seen it working properly, and the docs aren't helpful. That was yesterday, when I started to work on it today, I am noticing that it has stopped working entirely, and right now, the login popup is also not opening. I've tried `ccp#/` , tried `loginPopup: false` but no dice.
